WordPress/src/main/java/org/wordpress/android/ui/reader/services/update/ReaderUpdateLogic.java

```diff
@@ -193,6 +195,11 @@ public void run() {
                     }
                     // broadcast the fact that there are changes
                     EventBus.getDefault().post(new ReaderEvents.FollowedTagsChanged(true));
+                    // bump analytics
+                    shouldBumpTagsAnalytics = true;
+                }
+                if (shouldBumpTagsAnalytics) {
+                    ReaderTracker.trackFollowedTagsCount(subscribedTags.size());
```

Comment:
⚠️ I would prefer not to add Analytics tracking logic (which I would consider business logic) inside this data layer object. I believe you can accomplish the analytics tracking in some place that hooks into the `FollowedTagsChanged` event instead of doing all this logic inside the data fetching mechanism.